### PR TITLE
Small pathgasmeter cleanups

### PR DESCRIPTION
**Pull request by axic**
Split off #4779.

### CODE REVIEW

File: libevmasm/PathGasMeter.h [hunk 1]
```diff
@@ -57,6 +57,11 @@ class PathGasMeter
 
 	GasMeter::GasConsumption estimateMax(size_t _startIndex, std::shared_ptr<KnownState> const& _state);
 
+	static GasMeter::GasConsumption estimateMax(AssemblyItems const& _items, solidity::EVMVersion _evmVersion, size_t _startIndex, std::shared_ptr<KnownState> const& _state)
```
Comment (axic) [inline libevmasm/PathGasMeter.h]: Code formatting rule here?

Comment (leonardoalt) [reply libevmasm/PathGasMeter.h]: You could write
```
static GasMeter::GasConsumption estimateMax(
    AssemblyItems const& _items,
    solidity::EVMVersion _evmVersion,
    size_t _startIndex,
    std::shared_ptr<KnownState> const& _state
)
```
Or did you mean something else?

Comment (axic) [reply libevmasm/PathGasMeter.h]: Yep, wasn't sure what's the rule.

Comment (axic) [reply libevmasm/PathGasMeter.h]: Fixed.

File: libevmasm/GasMeter.cpp [hunk 2]
```diff
@@ -101,8 +101,8 @@ GasMeter::GasConsumption GasMeter::estimateMax(AssemblyItem const& _item, bool _
 			break;
 		case Instruction::KECCAK256:
 			gas = GasCosts::keccak256Gas;
-			gas += wordGas(GasCosts::keccak256WordGas, m_state->relativeStackElement(-1));
 			gas += memoryGas(0, -1);
+			gas += wordGas(GasCosts::keccak256WordGas, m_state->relativeStackElement(-1));
```
Comment (axic) [inline libevmasm/GasMeter.cpp:105]: This order is in line with the rest of the switch, for readability purposes.

**Review by chriseth — APPROVED** [review]

**Pull request merged**